src/AutoRest.CSharp/AutoRest.CSharp.csproj

```diff
@@ -17,8 +17,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.21.0-alpha.20211029.1" />
-    <PackageReference Include="Azure.Core.Experimental" Version="0.1.0-preview.16" />
+    <PackageReference Include="Azure.Core" Version="1.22.0-alpha.20211119.1" />
+    <PackageReference Include="Azure.Core.Experimental" Version="0.1.0-preview.18" />
```

Comment:
Sync version of WaitForCompletion method was not existed in the older Azure.Core version(`"1.21.0-alpha.20211029.1"`), so I had to update the core version in order to call `operation.WaitForCompletion();` for LLC LRO methods. [Here](https://github.com/Azure/autorest.csharp/blob/3c8aa4e060ca4729082a78bbd76ab43ade5facb2/test/TestProjects/LRO-LowLevel/Generated/LROServiceClient.cs#L112) is an example.